client/coral-admin/src/actions/settings.js

```diff
@@ -21,10 +23,16 @@ export const fetchSettings = () => dispatch => {
     });
 };
 
+// for updating top-level settings
 export const updateSettings = settings => {
   return {type: SETTINGS_UPDATED, settings};
 };
 
+// this is a nested property, so it needs a special action.
+export const updateWordlist = (listName, list) => {
```

Comment:
Good thinking @wyattjoh. List here is basically a data type. I love the ability to create granular api controls around adding/removing/counting etc... 'words'. 

Also +1 to snake_case_novella_variable_names